packages/block-library/src/pullquote/transforms.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * WordPress dependencies
+ */
+import { createBlock } from '@wordpress/blocks';
+import { create, join, split, toHTMLString } from '@wordpress/rich-text';
+
+const transforms = {
+	from: [
+		{
+			type: 'block',
+			isMultiBlock: true,
+			blocks: [ 'core/paragraph' ],
+			transform: ( attributes ) => {
+				return createBlock( 'core/pullquote', {
+					value: toHTMLString( {
+						value: join(
+							attributes.map( ( { content } ) =>
+								create( { html: content } )
+							),
+							'\u2028'
```

Comment:
I don't feel confident that this is the optimal way to do it, so I'll defer to @ellatrix for review. This includes the logic further down in the `'to'` transforms.

Reply:
This was actually the same as `Quote`. Didn't really dig into that. Seemed to me a Business Logic decision that has been made, to transform the first part only..

Reply:
You're right. That's good enough for me!